### PR TITLE
Renamed numeric ids to strings

### PR DESCRIPTION
**Pull request by Vindexus**
Issue: https://github.com/shawncplus/ranviermud/issues/339

### CODE REVIEW

File: bundles/ranvier-areas/areas/limbo/rooms.yml [hunk 1]
```diff
@@ -1,19 +1,19 @@
 - id: 1
```
Comment (seanohue) [inline bundles/ranvier-areas/areas/limbo/rooms.yml]: I would consider changing these room IDs as well, and if you want to be creative, titling them something other than "Test Room 1" and "Test Room 2". Thoughts?

File: bundles/ranvier-areas/areas/limbo/rooms.yml [hunk 2]
```diff
@@ -1,113 +1,116 @@
...
+  title: "White Room"
   description: "A featureless white room. A pitch black void in the shape of archway can be seen on the east side of the room."
-  npcs: ["limbo:1" ]
+  npcs: ["limbo:rat" ]
   items:
-    - id: "limbo:3"
+    - id: "limbo:woodenchest"
       respawnChance: 20
       replaceOnRespawn: true # when this chest respawns completely replace it so its contents get refreshed
   script: "1-test"
```
Comment (shawncplus) [inline bundles/ranvier-areas/areas/limbo/rooms.yml]: I suggest changing the script names to match the room/item ids in here and for items. The scripts live in `ranvier-areas/areas/limbo/scripts/ there

**Comment by Vindexus** [comment]
I'll push that soon. Are the docs in another repo?

**Comment by shawncplus** [comment]
Nope, the docs are just in the `docs/` directory

**Comment by Vindexus** [comment]
I updated the docs on this. I'm having a heck of a time getting mkdocs to work so I've just been double checking on GitHub. Let me know if I missed anything.

**Comment by Vindexus** [comment]
Does this PR need any changes?

**Pull request merged**